arapp.json

```
@@ -1,6 +1,6 @@
 {
-  "path": "contracts/DandelionVoting.sol",
-  "appName": "dandelion-voting.aragonpm.eth",
+  "path": "contracts/DisputableDandelionVoting.sol",
+  "appName": "dandelion-voting-disputable.aragonpm.eth",
```

Maybe we should be consistent and have `disputable` first 
```suggestion
  "appName": "disputable-dandelion-voting.aragonpm.eth",
```